.gitignore

```diff
@@ -0,0 +1,12 @@
+# Dependency directory
+node_modules/
+
+# Vim-related files
+*.swp
+*.vim
+
+# package lock file
+package-lock.json
+
+# bundle file
+home-app/bundle.js
```

Comment:
obsolete?

Reply:
I'm going to make this change in a separate PR for the sake of isolation